Rakefile

```diff
@@ -13,11 +14,75 @@ namespace :db do
 end
 
 namespace :web do
+  desc 'Create test kit pages from test kit metadata'
+  task :create_test_kit_pages do
+
+    TEST_KIT_DIR = 'web/_test_kits'
+    require 'inferno'
+    ENV['NO_DB'] = 'true'
+    Inferno::Application.start(:suites)
+
+    FileUtils.rm_rf(Dir.glob("#{TEST_KIT_DIR}/*.md"))
```

Comment:
Doing this without using `File.join(__dir__, ...)` makes me nervous.

Reply:
Yeah, good point.  I was looking at the constant and coudln't think of any way this could possibly mess up which is why I kept my initial quick and dirty solution instead of cleaning it up on the final pass before PR, but when rm_rf involved good to be a bit more careful in case someone messes with that constant in the future.  Fixed, I believe.

Reply:
Hmm, I forgot the `Dir.glob`.  Fixed again and I confirmed the delete works by adding in another 'excluded_test_kit' (but didn't commit that).